pkg/alertmanager/api.go

```diff
@@ -9,7 +9,7 @@ import (
 
 	"github.com/cortexproject/cortex/pkg/alertmanager/alerts"
 	"github.com/cortexproject/cortex/pkg/tenant"
-	"github.com/cortexproject/cortex/pkg/util"
+	util_log "github.com/cortexproject/cortex/pkg/util/log"
```

Comment:
I think the `pkg/util/log` should be renamed to `pkg/util/logutil` to avoid misunderstandings with `github.com/go-kit/kit/log`.

Reply:
that's what they were originally; @pstibrany requested that I rename them :see_no_evil:

Reply:
> I think the `pkg/util/log` should be renamed to `pkg/util/logutil` to avoid misunderstandings with `github.com/go-kit/kit/log`.

:) In my previous comment I actually suggested we do the opposite, to avoid repeating “util” in the package name. I don’t mind the conflicting names, and actually prefer “util_log” and “util_math” in imports (I wouldn’t mind to use that everywhere for consistency, but there is no need to).

Reply:
Didn't know you already discussed it. I missed that, sorry. Let's discuss it offline, but definitely not a good reason to block this PR.